src/SAWScript/Crucible/JVM/Override.hs

```diff
@@ -714,8 +692,7 @@ learnPointsTo opts sc cc spec prepost pt = do
            case toJVMType (Cryptol.evalValType mempty ety) of
              Nothing -> fail "jvm_array_is: invalid element type"
```

Comment:
Is this failure hard to include in the structured error type you created?

Reply:
Similarly, this should really be a `panic` at this point, as the types should have been checked during the `JVMSetup` phase.

Reply:
Sounds good.